public/index.html

```diff
@@ -0,0 +1,493 @@
+    </section>
+    <section class="newspaper-section fourteen-skew">
+      <h1>AppBox Featured In ...<span class="accent-bar-C"></span></h1>
+        <ul class="newspaper-list">
+          <li>
+            <a href="http://www.nytimes.com/2009/09/10/technology/personaltech/10app.html?_r=0"></a>
+          </li>
+          <li>
+            <a href=""></a>
+          </li>
+          <li>
+            <a href=""></a>
+          </li>
+          <li>
+            <a href=""></a>
+          </li>
+          <li>
+            <a href=""></a>
+          </li>
+          <li>
+            <a href=""></a>
+          </li>
+          <li>
+            <a href=""></a>
+          </li>
+          <li>
+            <a href=""></a>
+          </li>
+        </ul>
+    </section>
+		<section id="support-nav" class="app-support fifteen-skew clearfix">
+      <h1>App Support<span class="accent-bar-C"></span></h1>
+      <ul>
+        <li class="clearfix">
+          <span class="icon-location sm-circular-eclipse-border"></span>
+          <h3>Address</h3>
+          <address>121 King Street, Melbourne<br/>Victoria, Australia 3000</address>
+        </li>
+        <li class="clearfix">
+          <span class="icon-phone sm-circular-eclipse-border"></span>
+          <h3>Free Helpline</h3>
+          <a href="tel:+001800345333700">(001) 800 345 333700</a>
+          <a href="tel:+002600345333700">(002) 600 345 333700</a>
+        </li>
+        <li class="clearfix">
+          <span class="icon-mail sm-circular-eclipse-border"></span>
+          <h3>Email</h3>
+          <a href="mailto:info@appbox.com">info@appbox.com</a>
+          <a href="mailto:feeback@appbox.com">feedback@appbox.com</a>
+        </li>
+      </ul>
+
+      <form id="helpform" class="fullform">
+        <p>Nullam id odio quis risus placerat aliquam. Cum sociis natoque penatibus et magnis dis part urient montes nascetur ridiculus mus curabitur tellus lacus.</p>
+        <input type="text" placeholder=" Full Name" name="help-name" id="help-name">
+        <input type="email" placeholder=" Email" name="help-email" id="help-email">
+        <textarea name="message" form="helpform" rows="8">Message</textarea>
+				<button type="submit" class="sm-button-eclipse-border jiggle-effect">SEND</button>
+      </form>
+    </section>
+    <div class="social-section seventeen-skew">
+      <section class="info">
+        <div class="triangle">
+          <a href="#home-nav" class="arrow"></a>
+        </div>
+        <span class="logo">&nbsp;</span>
+        <p>Lorem ipsum dolor amet consecte adipisicing elit sed do eiusmod tempor incididunt ut labore dolore magna aliqua. Ut enim ad minim veniam quis nostrud exercitation ullamco.</p>
+        <p>Laboris nisi dut aliquip ex ea commodo. conse quat. Duis aute irure dolor reprehenderit</p>
+      </section>
+      <section class="blogroll">
+        <h1>Latest From Blog<span class="accent-bar-L"></span></h1>
+          <ul>
+            <a href="#">
```

Comment:
`a` tags are not allowed as child elements of `ul`. Start with the `li` and then use the `a`.Use `display: block` to fill the space